test/models/supervisor_run_test.rb

```diff
@@ -138,4 +141,26 @@ def setup
     assert_equal @supervisor_script.read_log, "Unable to load log file: #{FILE_PATH}"
   end
 
+  test "proper behaviour of stop method" do
+    @supervisor_script.pid = PID
+    @supervisor_script.is_running = true
+    @supervisor_script.expects(:check).returns(true).twice
+    Process.expects(:kill).with('INT', PID)
+    Process.expects(:kill).with('TERM', PID)
+
+    @supervisor_script.stop
+
+    assert_equal false, @supervisor_script.is_running
+  end
+
+
+  test "proper behaviour of destory method" do
```

Comment:
W tym pliku testów jedynie proper behaviour of destroy metod wymaga połączenia z bazą - jest to katalog "models" - można ten test przenieść do integracyjnych, żeby utrzymywać konwencję, gdzie testy w model nie łączą się z bazą.